### ISSUE TITLE
example code for train a logistic regression using tensorframes

### ISSUE THREAD

**Issue by yupbank** [databricks/tensorframes#103]
So it is not very clear how a `tf.Variable` works with tensorframes.
especially with trainable executions.

**Comment by wuciawe** [comment]
hi @yupbank did you figure out how to train a logistic regression with tensorframes?

**Comment by yupbank** [comment]
not really, well, i can write a logistic regression with tensorframes for sure but then i'll need to do all the math

and tensorframes are mostly for inferencing

**Comment by wuciawe** [comment]
Is it model parallel, or still data parallel such as with dataset? Does tensorframes support parameter server such as distributed tensorflow? I have sparse features with very high dimension. So I'd like to have data parallel with spark and model parallel with tensorflow.
